docs/content/policy-testing.md

```diff
@@ -50,23 +50,24 @@ To test this policy, we will create a separate Rego file that contains test case
 **example_test.rego**:
 
 ```live:example/test:module:read_only
-package authz
+package authz_test
 import rego.v1
+import authz
```

Comment:
While we are here, adding some newlines here to align with `opa fmt` would be good.

I'd change:
```rego
package authz_test
import rego.v1
import authz
```

To
```rego
package authz_test

import rego.v1

import authz
```

And make it so for all of the examples in this section.